RealEstate/RealEstate/Pages/Index.cshtml.cs

```diff
@@ -26,19 +26,38 @@ public void OnGet()
 
             using (var webClient = new WebClient())
             {
-                // Grab our JSON text. 
-                var houseJSON = webClient.DownloadString("https://data.cityofchicago.org/resource/s6ha-ppgi.json");
+                
+                string Chicago_data = string.Empty;
+
+                try
+                {
+                    Chicago_data = webClient.DownloadString("https://data.cityofchicago.org/resource/s6ha-ppgi.json");
+                }
+                catch (Exception e)
+                {
+                    Console.WriteLine("Exception while calling API", e);
+                }
 
                 // Convert raw text to list.
-                List<Houses> houseCollection = Houses.FromJson(houseJSON);
+                List<Houses> houseCollection = Houses.FromJson(Chicago_data);
```

Comment:
Descriptive variable names are good, but....
- Chicago_data is equally as ambiguous as houseJSON.
- Variable names should not include _, unless it is a constant.